pkg/runner/local_common.go

```diff
@@ -80,7 +80,7 @@ func localCommonHealthcheck(ctx context.Context, hh *healthcheck.Helper, cli *cl
 		healthcheck.StartContainer(ctx, ow, cli, &docker.EnsureContainerOpts{
 			ContainerName: "testground-influxdb",
 			ContainerConfig: &container.Config{
-				Image: "bitnami/influxdb",
+				Image: "library/influxdb",
```

Comment:
Probably a good idea to specify the exact tag we are using now.

Reply:
Yeah, should be fine because 1.8+ is good for us, but can do.